README.md

```diff
@@ -8,7 +8,7 @@ _export:
     repositories:
       - https://DeNA.github.io/analytics-maven-repo
     dependencies:
-      - com.dena.digdag:digdag-operator-bq-wait:0.1.0
+      - com.dena.digdag:digdag-operator-bq-wait:0.1.1
```

Comment:
@rhase 
it might be incorrect procedure that i versioned up to 0.1.1 before publishing repo.

now i got this error.
```
java.lang.RuntimeException: org.eclipse.aether.resolution.DependencyResolutionException: Could not find artifact com.dena.digdag:digdag-operator-bq-wait:jar:0.1.1 in central (https://repo1.maven.org/maven2/)
````

please publish latest version to repo. it's very helpful!

Reply:
Yes! Actually right now I'm working on it.
(I should've publish before merging pr... )
I'll let you know when I published 0.1.1

Reply:
@kozuke 
I published v0.1.1 and confirmed digdag works all right.
Please check it out.

Reply:
@rhase 
worked fine! Thx for your quick response and delivery.